### ISSUE TITLE
letsencrypt should run in non-interactive mode

### ISSUE THREAD

**Issue by bitmage** [ployst/docker-letsencrypt#18]
I'm testing out the new release: `0.3.0`.  I ran `./refresh_certs.sh` and it asked me to share my email:

```
-------------------------------------------------------------------------------
Would you be willing to share your email address with the Electronic Frontier
Foundation, a founding partner of the Let's Encrypt project and the non-profit
organization that develops Certbot? We'd like to send you email about EFF and
our work to encrypt the web, protect its users and defend digital rights.
-------------------------------------------------------------------------------
(Y)es/(N)o:
```

This is going to cause scheduled jobs to fail.  Recommend using the `--non-interactive` flag, [as described here](https://github.com/certbot/certbot/issues/804#issuecomment-215406768).

**Comment by czerasz** [comment]
I just run into this issue while testing my cron job.
I ended up using the `-n` flag because `--non-interactive` wasn't available in `certbot 0.14.0`:

```
$ certbot --help | grep 'interactive'
  --manual          Obtain certs interactively, or using shell script hooks
   -n               Run non-interactively
```

**Comment by txels** [comment]
This is probably fixed in #21 where the flag `--no-eff-email` was added. Please reopen if this is not the case.
